ietf/doc/views_conflict_review.py

```diff
@@ -455,13 +455,19 @@ def start_review_sanity_check(request, name):
 
     return doc_to_review
 
+
 def build_notify_addresses(doc_to_review):
     # Take care to do the right thing during ietf chair and stream owner transitions
     notify_addresses = []
-    notify_addresses.extend([r.formatted_email() for r in Role.objects.filter(group__acronym=doc_to_review.stream.slug, name='chair')])
+    if doc_to_review.stream.slug not in ['ise', 'irtf']:
+        notify_addresses.extend(
+            [r.formatted_email() 
+             for r in Role.objects.filter(group__acronym=doc_to_review.stream.slug, name='chair')]
```

Comment:
Does this match the addresses now being filtered out from the notify list? Here, it's addresses for 'chair' Roles. The filtered items are 'stream_managers' from Recipients. Do these match up in the right way?